config/puma.rb

```diff
@@ -32,3 +33,15 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+before_fork do
+  require 'puma_worker_killer'
+  PumaWorkerKiller.config do |config|
+    config.ram           = ENV.fetch('PUMA_RAM') { 550 } # mb
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [Sider](https://sider.review) -->